### DISCUSSION TITLE
Can't restore table schema by clickhouse-backup after drop table statement

### DISCUSSION THREAD

**Discussion by urix24** [Altinity/clickhouse-backup#611] [Q&A] [answered]
Hi Everyone!

clickhouse version 23.1.3.5 (Also tried on early releases)
clickhouse-backup version 2.1.3
I Made backup from inside docker container:

`./clickhouse-backup create --tables=temp3.test3 test3.bkp`

Then dropped the table:

`drop table temp3.test3`

And tried to restore it:

`./clickhouse-backup restore --tables=temp3.test3  test3.bkp`

Got the following error:

`error can't create table temp3.test3: code: 57, message: Directory for table data store/517/517ab587-39c2-464e-8cc3-61e4987c7306/ already exists after 1 times, please check your schema dependencies`

Then I created table by:

```
CREATE TABLE IF NOT EXISTS temp3.test3
                                   (
                                     `date` Datetime,
                                     `user_id` String,
                                     `pageviews` Int32
                                   )
                                   ENGINE = MergeTree()
                                   PARTITION BY toStartOfHour(date)
                                   ORDER BY (date)
```

And restored table without shema, only data, it works and if I truncate the table not drop it works to:

`./clickhouse-backup restore --tables=temp3.test3 --data test3.bkp`

Where is my mistake ? Thanks.

**Comment by Slach** [comment, marked as answer]
Your database engine is Atomic
you need to use
```sql
drop table temp3.test3 SYNC
```
instead of just drop table

or use 
`clichouse-backup- restore --drop --tables=temp3.test3 test3.bkp`

**Reply by urix24** [reply]
Thank you.
It works now with 

`drop table temp3.test3 SYNC`
